Sources/Extensions/CollectionViewManager+UICollectionViewDataSource.swift

```diff
@@ -16,6 +16,7 @@ extension CollectionViewManager: UICollectionViewDataSource {
         let cellItem = self.cellItem(for: indexPath)!
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: cellItem.reuseType.identifier, for: indexPath)
         cellItem.configure(cell)
+        cellItem.cachedSize = nil
```

Comment:
Why is it getting reset here?
Maybe it would be a good idea to provide more flexible controls for cache invalidation?
I would actually be ok with caching behavior being explicit, so developer will need to specify what / when should be cached or invalidated.